evoting/protocol/shuffle.go

```diff
@@ -130,6 +130,17 @@ func (s *Shuffle) HandlePrompt(prompt MessagePrompt) error {
 		}
 
 		a, b := lib.Split(ballots)
+		// Protect from missing input.
+		for i := range a {
+			if a[i] == nil {
+				a[i] = cothority.Suite.Point()
```

Comment:
Here, and `b[i]`, is it OK to have a somewhat undefined point? As per the comment above, it's not guaranteed that it's `Null()`.